src/main/java/org/humancellatlas/ingest/schemas/web/SchemaController.java

```diff
@@ -0,0 +1,52 @@
+package org.humancellatlas.ingest.schemas.web;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.humancellatlas.ingest.schemas.Schema;
+import org.humancellatlas.ingest.schemas.SchemaService;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.rest.webmvc.PersistentEntityResourceAssembler;
+import org.springframework.data.rest.webmvc.RepositoryRestController;
+import org.springframework.data.web.PagedResourcesAssembler;
+import org.springframework.hateoas.ExposesResourceFor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Created by rolando on 19/04/2018.
+ */
+@RepositoryRestController
+@ExposesResourceFor(Schema.class)
+@RequiredArgsConstructor
+public class SchemaController {
+    private final @NonNull SchemaService schemaService;
+    private final @NonNull PagedResourcesAssembler pagedResourcesAssembler;
+
+
+    @RequestMapping(path = "/schemas/update", method = RequestMethod.POST)
+    ResponseEntity<?> triggerSchemasUpdate() {
```

Comment:
Endpoints like `/schemas/update` are more RPC-style than RESTful (unless "update" here is used as a noun). It's not necessarily bad, but seeing as we're trying to adhere to HATEOAS in our API's we probably want to be as RESTful as possible. A more RESTful endpoint I think would be `/schemas` using HTTP PUT or PATCH when updating, which also means the request has to be sent with some data. I noticed that this end point also supports POST action but it doesn't expect any payload from the client.

Reply:
This method is intended to be a RPC to instruct the core to scrape the schema bucket. Could paramaterize it but there isn't any reason to at the moment